content/blog/2020-03-dplyr-1-0-0-new-verb-features/index.Rmarkdown

```diff
@@ -0,0 +1,231 @@
+---
+title: 'dplyr 1.0.0: new verb features'
+author: Hadley Wickham
+date: '2020-03-20'
+slug: dplyr-1-0-0-new-verb-features
+categories:
+  - package
+tags:
+  - dplyr
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+As we've mentioned, [dplyr 1.0.0](https://www.tidyverse.org/blog/2020/03/dplyr-1-0-0-is-coming-soon/) is on its way. Today, we've started the official release process by notifying maintainers of packages that have problems with dplyr 1.0.0, and we're planning for a CRAN release on May 1.
+
+In this post, I'll introduce you to the major changes in existing dplyr verbs, along with couple of new verbs. I'll give you updates on `summarise()`, `select()` + `rename()`, `mutate()`, and the join functions.
+
+```{r setup, message = FALSE}
+library(dplyr)
+```
```

Comment:
Maybe mention how to get the dev version here? Or, perhaps better, just reveal the dplyr version that's just been attached. In any case, somehow make it crystal clear that other folks, who've got current CRAN dplyr, will NOT see these new features if they run the code in the post.

Reply:
Good point, thanks!